src/main/java/gyro/google/dns/PolicyResource.java

```diff
@@ -0,0 +1,276 @@
+                policy.setAlternativeNameServerConfig(alternativeNameServerConfig == null
+                    ? Data.nullOf(PolicyAlternativeNameServerConfig.class)
+                    : alternativeNameServerConfig.copyTo());
+            } else if (changedFieldName.equals("enable-inbound-forwarding")) {
+                policy.setEnableInboundForwarding(getEnableInboundForwarding());
+            } else if (changedFieldName.equals("enable-logging")) {
+                policy.setEnableLogging(getEnableLogging());
+            } else if (changedFieldName.equals("network")) {
+                List<DnsPolicyNetwork> networks = getNetwork();
+                policy.setNetworks(networks == null
+                    ? Data.nullOf(ArrayList.class)
+                    : networks.stream()
+                        .map(DnsPolicyNetwork::copyTo)
+                        .collect(Collectors.toList()));
+            }
+        }
+        Dns client = createClient(Dns.class);
+        PoliciesPatchResponse response = client.policies().patch(getProjectId(), getName(), policy).execute();
+        refreshFrom(response.getPolicy());
+    }
+
+    @Override
+    public void doDelete(GyroUI ui, State state) throws Exception {
+        Dns client = createClient(Dns.class);
+        client.policies().delete(getProjectId(), getName()).execute();
+    }
+
+    @Override
+    public void copyFrom(Policy model) {
+        PolicyAlternativeNameServerConfig policyAlternativeNameServerConfig = model.getAlternativeNameServerConfig();
+
+        if (policyAlternativeNameServerConfig != null) {
+            DnsPolicyAlternativeNameServerConfig alternativeNameServerConfig = getAlternativeNameServerConfig();
+            alternativeNameServerConfig.copyFrom(policyAlternativeNameServerConfig);
+        }
+        setDescription(model.getDescription());
+        setEnableInboundForwarding(model.getEnableInboundForwarding());
+        setEnableLogging(model.getEnableLogging());
+        setId(model.getId().toString());
+        setName(model.getName());
+        List<PolicyNetwork> networks = model.getNetworks();
+
+        if (networks != null && !networks.isEmpty()) {
+            setNetwork(networks
+                .stream()
+                .map(network -> {
+                    DnsPolicyNetwork diffablePolicyNetwork = Optional.ofNullable(
+                        getNetwork())
+                        .orElse(Collections.emptyList())
+                        .stream()
+                        .filter(e -> e.isEqualTo(network))
+                        .findFirst()
+                        .orElse(newSubresource(DnsPolicyNetwork.class));
+                    diffablePolicyNetwork.copyFrom(network);
+                    return diffablePolicyNetwork;
+                })
+                .collect(Collectors.toList()));
+        }
+    }
+
+    private boolean refreshFrom(Policy managedZone) {
+        copyFrom(managedZone);
+        return true;
+    }
```

Comment:
This method does not serve any special purpose and should be removed and the `copyFrom` method should be used in all the places this method is used.